prydaen-favors.lic

```diff
@@ -0,0 +1,114 @@
+=begin
+    Author: Damiza Nihshyde
```

Comment:
Please move this info to elanthipedia and link to it here

Reply:
Doh!  I meant to erase that.  My apologies.